grafana/setup_grafana.go

```diff
@@ -43,19 +43,19 @@ type grafanaConfig struct {
 func main() {
 
 	envParams := map[string]string{
-		"grafana_user":              "admin",
```

Comment:
Backward capability may be broken. We should just add `gf_security_admin_user` and `gf_security_admin_user` to the map instead of replacing `grafana_user` and `grafana_passwd`.